test/lib.sh

```diff
@@ -59,3 +61,72 @@ EOF
 hr() {
     echo "========================="
 }
+
+## compare FOLDER LOG_FILE
+#
+# Check FOLDER is properly sync'ed, determining the reference base on FOLDER 
+#
+# :param FOLDER: An existing directory sync'ed by boilerplate and to be checked
+# :param LOG_FILE: The log file used to aggregate the output of the `diff` calls
+compare() {
+	if [ $1 = "_data" ] ; then
+		if [ ! -f $repo/boilerplate/_data/last_boilerplate_commit ] ; then
```

Comment:
Sorry, I just noticed you've used this `$repo` variable in several places throughout this file, but I don't see anywhere it's defined.

Reply:
Okay, I see it's being set in the test case itself. Needs to be a parameter instead.